networkx/algorithms/community/__init__.py

```diff
@@ -1,9 +1,10 @@
 """Functions for computing and measuring community structure.
 
-The functions in this class are not imported into the top-level
-:mod:`networkx` namespace. You can access these functions by importing
-the :mod:`networkx.algorithms.community` module, then accessing the
-functions as attributes of ``community``. For example::
+.. warning:: The functions in this class are not imported into the top-level
+:mod:`networkx` namespace. 
+
+They can be imported using the :mod:`networkx.algorithms.community` module,
+then accessing the functions as attributes of ``community``. For example::
```

Comment:
Users can now access these function via e.g. `nx.community.girvan_newman` without a new import statement.
Should we advertise that more in places like this? 

We should go through the codebase and update claims like these now that we are using lazy_loading. But that (and maybe this comment) belong in a different PR.

Reply:
This change does belongs to a different PR. I'll go ahead and reverse it